DeAceroDLabs/marketplace-frontend
# Feature/dmp 640 Google maps api

## What is done

[[DMP-640] Google maps api](https://deacero.atlassian.net/browse/DMP-640)

## How is done

- Created a `Map` component that will be used as base for when a map is needed.
- Created a `MapField` component that will be used on signup form.
- Added REACT_APP_GOOGLE_MAPS_API_KEY to env file locally and on the vm. (ask Diana for the value).

## How to test

1. Run the project via `npm start`
2. Click on Registrarse.
3. Fill first form.
4. You should see the map on second form.
